lib/microdata.ex

```diff
@@ -114,7 +114,7 @@ defmodule Microdata do
   ```
   """
 
-  @default_strategies [Microdata.Strategy.HTMLMicrodata, Microdata.Strategy.JSONLD]
+  @default_strategies [Microdata.Strategy.JSONLD, Microdata.Strategy.HTMLMicrodata]
```

Comment:
At least where we're at today, this is going to break as many sites as it fixes (as, for example, the seriouseats duck recipe whose test goes red on this commit). What we need to be doing in this PR is to merge the two approaches instead of preferring one to the other; the order that they're interrogated is of no consequence.

Reply:
Yeah, figured that was where the red was coming from. +1